lib/beaker/subcommands.rb

```diff
@@ -0,0 +1,22 @@
+require "thor"
+require "fileutils"
+require_relative "shared/subcommands_util"
+
+module Beaker
+  class Subcommands < Thor
+    include Beaker::Shared::SubcommandsUtil
+
+    desc "init", "Initialises the beaker test environment"
+    long_desc <<-LONGDESC
+      Initialises a beaker environment using a combination of environment variables and options
```

Comment:
should we consult someone from ux on the wording of the short and long descriptions here? i just winged it here to have something to print out